service/pipelines_test.go

```diff
@@ -210,6 +210,9 @@ func TestBuildPipelines(t *testing.T) {
 			})
 			assert.NoError(t, err)
 
+			pipelines, ok := pipelinesInterface.(*builtPipelines)
+			require.True(t, ok)
```

Comment:
assert.IsType(t, &builtPipelines{}, pipelinesInterface)

Reads easier what you are checking.

Reply:
Fixed